lib/logstash/inputs/eventlog.rb

```diff
@@ -21,120 +22,62 @@ class LogStash::Inputs::EventLog < LogStash::Inputs::Base
   default :codec, "plain"
 
   # Event Log Name
-  config :logfile, :validate => :array, :default => [ "Application", "Security", "System" ]
+  config :logfile, :validate => :string, :validate => [ "Application", "Security", "System" ], :default => "Application"
```

Comment:
Should we add to the documentation how to grant access to read from the Security Logs without being Administrator?

https://social.technet.microsoft.com/forums/windowsserver/en-US/d2f813db-6142-4b5b-8d86-253ebb740473/easy-way-to-read-security-log